src/OrchardCore.Modules/OrchardCore.Media/OrchardCore.Media.csproj

```diff
@@ -3,6 +3,12 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Why?

Reply:
Hmmm ... not sure why I put that there. I recognize that pattern from our project and it shouldn't be there ... I'll remove it.

>             Why?